.buildkite/pipeline-default.yml

```diff
@@ -14,19 +14,19 @@ steps:
     agents:
       queue: builders
   - wait
-  - name: ":go: 1.13 test - %n"
+  - name: ":go: 1.15 test - %n"
```

Comment:
In this file there are tasks for 1.13 and 1.14. Those tasks are not exactly the same. 1.13 should become 1.14 and 1.14 should become 1.15.